### ISSUE TITLE
Using {@inheritdoc} in apidoc

### ISSUE THREAD

**Issue by vladim1** [yiisoft/yii2#7789]
I am using {inheritdoc} inline tag to inherit the parent description. 

``` php
    /**
     * {@inheritdoc}
     */
    public function beforeAction($action)
    {
    ...
    }
```

After generating documentation with yii2-apidoc I am expecting to get parent description in the description section, but I am getting `{@inheritdoc}` as a text.

```
vendor/bin/apidoc api /var/www/project.com /home/user/doc
```

Is {@inheritdoc} support is turned off or I have missed something?

UPD: it looks like the reason `{inheritdoc}` doesn't work is because apidoc doesn't know where the parent class is situated. 
In my case the parent class is `yii\web\AssetBundle`. 
I think so, because the apidoc doesn't create the link to `yii\web\AssetBundle`. I am using advanced application template. How can I force apidoc to analyze vendor folder. As was mentioned before I specified the project folder(which includes frontend,backend, vendor).

**Comment by samdark** [comment]
It's supported but w/o `{` and `}`.

**Comment by vladim1** [comment]
@samdark 
I have updated my post just before your answer. Please, see UPD section in the first post.

In my case {@inheritdoc} doesn't work without ```{``` and ```}``` either. But in this case the generated description is blank.

**Comment by cebe** [comment]
To include vendor classes in the api documentation you have to add their path to the input directory. You can have many input directories separated by comma:

```
./apidoc api app/,vendor/yiisoft,...
```

**Comment by vladim1** [comment]
I have tried to include ```vendor/yiisoft``` dir as an input directory. From the project dir:
```
vendor/bin/apidoc api backend,frontend,vendor/yiisoft,common,console /home/user/doc
```
The result is pretty weird. 
First of all the apidoc has created all web pages. But in the index.html file I can see only the menu for yii classes (from vendor folder). 
```
file:///home/user/doc/index.html
```
And there are no classes from backend, console, frontend, common folders. 
When I put the full path to the generated page, like
```
file:///home/user/doc/backend-assets-appasset.html
```
I see all the packages (common, frontend, backend, console) except the classes from vendor folder. Why is it happening?
And the inheritance still doesn't work. I used ```@inheritdoc``` without curly brackets. The result is blank. When I user ```{@inheritdoc}``` the result is ```{@inheritdoc}``` as a text in the description.
